src/modules/faucet/utils/index.ts

```diff
@@ -58,6 +58,10 @@ export const sendFaucet = async ({
     const dripAmount = ethers.utils.parseUnits(amount.toString(), ASTAR_TOKEN_DECIMALS).toString();
     const result = await astarApi.sendTokenTo({ to: address, dripAmount: new BN(dripAmount) });
 
+    if (network === Network.shiden) {
+        await astarApi.connectTo(Network.shibuya);
```

Comment:
@hoonsubin 
Yes, so here the API instance connects to 'shibuya'. Does this look good to you?